concent_api/api_testing_common.py

```diff
@@ -72,6 +74,14 @@ def assert_condition(actual, expected, error_message = None):
         raise TestAssertionException(message)
 
 
+def compare_lists_regardless_of_order(actual: List[str], expected: List[str]) -> bool:
```

Comment:
do you really need all those assertions?

Reply:
I know you hate assertions :D. In this case... I think them useful, they are 'sanity check'. In this place I expect list of strings and I check if it is. I think this asserts will never fail, but in this case they can save lot of debugging time. I would leave them here, it is test-util, but if you think they should be deleted I will :)

Reply:
I think this is an overkill - you have simple function, which's core is basically one line. Piling so many asserts makes it harder to get. I really see no point in adding theme here.

Reply:
done, deleted